Source/EasyNetQ/Consumer/DefaultConsumerErrorStrategy.cs

```diff
@@ -167,7 +167,7 @@ private string DeclareErrorExchangeWithQueue(IModel model, MessageReceivedInfo r
         var errorExchangeType = conventions.ErrorExchangeTypeConvention();
         var errorQueueName = conventions.ErrorQueueNamingConvention(receivedInfo);
         var errorQueueType = conventions.ErrorQueueTypeConvention();
-        var routingKey = receivedInfo.RoutingKey;
+        var routingKey = conventions.ErrorExchangeRoutingKeyConvention(receivedInfo);
```

Comment:
```
model.BasicPublish(errorExchange, receivedInfo.RoutingKey, errorProperties, message.Memory);
```


Should routing key be modified somehow in this line above?

Reply:
Changed the logic with a new method to consider the routing key from conventions for error queue. Please check now

Reply:
I think should be done as the basicpublish will use default routing key if not changed for publish instead of one set in conventions

Reply:
It depends on use case.

You mentioned that you are not happy that so many binding are created. To reduce the count, you might be "wildcarding" them: just using `#`/`prefix.#`/`prefix.*` as a binding key for error queue.

That's why I am not sure that the routing key of a message should be modified.

Reply:
As for now, I've rolled it back. Let's discuss it separately.

Reply:
sure, i guess then binding to error exchange happen with custom routing key but publish happens from the default routing key of received info. Have to check on how this will work.

Any how - thanks for merging it to master - i will check the 8.0.0.beta2 version with custom routing key and will keep you posted

Reply:
Also - you mentioned that - "you might be "wildcarding" them: just using #/prefix.#/prefix.* as a binding key for the error queue" - may i know where should this wildcarding be mentioned. Is it some parameter available on the conventions for the error queue?